test/parallel/test-force-repl.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
-const common = require('../common');
+require('../common');
 const assert = require('assert');
 const spawn = require('child_process').spawn;
 
 // Spawn a node child process in interactive mode (enabling the REPL) and
-// confirm the '> ' prompt is included in the output.
+// confirm the '> ' prompt and welcome message is included in the output.
 const cp = spawn(process.execPath, ['-i']);
 
 cp.stdout.setEncoding('utf8');
 
-cp.stdout.once('data', common.mustCall(function(b) {
-  assert.strictEqual(b, '> ');
-  cp.kill();
-}));
+let out = '';
+cp.stdout.on('data', (d) => {
+  out += d;
+});
+
+cp.stdout.on('end', () => {
+  assert(out.startsWith('Welcome to Node.js'));
+  assert(out.includes('Type ".help" for more information.'));
+  assert(out.endsWith('> '));
+  console.log('ok');
+});
```

Comment:
Please use `common.mustCall()` here. Otherwise there is no actual guarantee that the best passed.